config/locales/en.yml

```diff
@@ -69,6 +69,8 @@ en:
   reset_tour: "Reset tour"
   reset_tour_success: "Tour was successfully reset"
   reset_tour_fail: "An error has ocurred, please try again"
+  stories_destroy_success: "Stories were successfully destroyed"
+  stories_destroy_fail: "Stories were not successfully destroyed"
```

Comment:
I think it's better, the failure messages, be expressed in another way, without 'successfully' word in the sentence. Something like "Stories couldn't be destroyed due some validation errors"

Reply:
Of course looks very weird having "successfully" in a failure message, I don't know how I don't think in it. Thanks!